prototypes/multi-module-prototype/examples/multi-module-example/ModulesPrototype/Program.cs

```diff
@@ -70,22 +72,45 @@ public static async Task Main(string[] args)
         var loader = factory.Create(catalogue);
         var moduleCounter = new AsyncCountdownEvent(0);
 
-        var processExplorer = new HostBuilder()
-            .ConfigureLogging(l => l.AddConsole().SetMinimumLevel(LogLevel.Debug))
-            .ConfigureServices(
-             (context, services) => services
-                 .ConfigureSubsystemLauncher(loader.RequestStartProcess, loader.RequestStopProcess, CreateLaunchRequest, CreateStopRequest)
-                 .AddProcessExplorerWindowsServerWithGrpc(pe => pe.UseGrpc())
-                 .Configure<ProcessExplorerServerOptions>(op =>
-                 {
-                     op.Port = 5056;
-                     op.MainProcessId = Process.GetCurrentProcess().Id;
-                     op.EnableProcessExplorer = true;
-                 }))
-            .Build();
-
-        await processExplorer.StartAsync(cts.Token);
-        var infoAggregator = processExplorer.Services.GetRequiredService<IProcessInfoAggregator>();
+        //var processExplorer = new HostBuilder()
+        //    .ConfigureLogging(l => l.AddConsole().SetMinimumLevel(LogLevel.Debug))
+        //    .ConfigureServices(
+        //     (context, services) => services
+        //         .ConfigureSubsystemLauncher(loader.RequestStartProcess, loader.RequestStopProcess, CreateLaunchRequest, CreateStopRequest)
+        //         .AddProcessExplorerWindowsServerWithGrpc(pe => pe.UseGrpc())
+        //         .Configure<ProcessExplorerServerOptions>(op =>
+        //         {
+        //             op.Port = 5056;
+        //             op.MainProcessId = Process.GetCurrentProcess().Id;
+        //             op.EnableProcessExplorer = true;
+        //         }))
+        //    .Build();
+
+        var processExplorer = WebApplication.CreateBuilder(args);
+        processExplorer.Services.AddGrpc();
+        processExplorer.Services.AddCors(o => o.AddPolicy("AllowAll", builder =>
+        {
+            builder.AllowAnyOrigin()
```

Comment:
Also this kind of configuration should be contained inside the server (ASP.NET package) as we discussed earlier. Right now as we discussed it's okay to put it here.